packages/interbit-ui-components/src/tests/uiKit.test.js

```diff
@@ -30,6 +32,80 @@ const countChildren = (parent, child, count) => {
   expect(wrapper.find(child).length).toBe(count)
 }
 
+describe('<AppBucket/>', () => {
```

Comment:
When I run the tests, there is a warning about `history.push` being a missing required prop. Please update the test so that we don't have warnings.